app/(farmer)/dashboard/@logging/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { jsonApiFetch } from "@/lib/axios";
+import { getCookies } from "@/lib/cookie";
+import { FarmerJournal } from "@/types/content";
+import { Node } from "@/types/highLevel";
+import Link from "next/link";
+
+type Result = Pick<Node<FarmerJournal>, "created">
```

Comment:
```suggestion
type lastRecordTimestampResult = Pick<Node<FarmerJournal>, "created">
```

Should we have more descriptive result names? Or should

Reply:
I think no need to be descriptive for internal types. If I need to export the types, then yes.

Reply:
Hmm, okay, I think that makes sense.

I think it comes down to style: while the descriptive variable names make it easier for others to read your code, it also takes more time (and adds a form of coupling).

So no worries.